graylog2-web-interface/src/components/login/LoginChrome.tsx

```diff
@@ -54,6 +54,7 @@ const NotificationsContainer = styled.div`
   position: fixed;
   top: 0;
   margin-top: 5px;
+  width: 75%;
```

Comment:
I just saw that when we define the `width` this way, we depend on the width of the left col (for the form) to always be `33%`. Once the left column width changes, it can look like this:
![image](https://user-images.githubusercontent.com/46300478/204767375-756bb400-1f16-4996-aee1-595762389281.png)

Instead we can use `width: 100%` and `position: absolute`:
![image](https://user-images.githubusercontent.com/46300478/204767446-69632744-fd11-41c9-9ce6-03e3eec91af9.png)